.github/workflows/docs-ci.yaml

```diff
@@ -0,0 +1,29 @@
+name: docs-ci
+on:
+  push:
+    branches:
+      - master
```

Comment:
Doesn't this mean we're only showing this staged information when the PR is merged? Shouldn't this be setup to allow people to visualize their changes on a branch _before_ a merge?

Reply:
resolved in Slack, this is not intended to be a final solution for staging/publishing